test/test_torch.py

```diff
@@ -3652,6 +3652,12 @@ def test_norm_fastpaths(self):
         expected = torch.pow(x.pow(3).abs().sum(1), 1.0 / 3.0)
         self.assertEqual(result, expected)
 
+        inputValuesFP32 = torch.tensor([[1, 2, 3], [-1, 1, 4]], dtype=torch.float)
+        inputValuesBF16 = inputValuesFP32.bfloat16()
+        precision_3dps = 0.004
```

Comment:
Manually setting is bad, use @precisionOverride for this. Also, I'm surprised the error is so large.

Reply:
The added code is trying to compare the accuracy between BFloat16 and Float32. But the decorator of `@precisionOverride` will be bound to only one special data type at one time. It does not support test cases with mixed data types. In addition, I think the accuracy is under the normal level for low precision. The precision accuracy of `torch.half ` for [test_logspace](https://github.com/pytorch/pytorch/blob/master/test/test_torch.py#L10538) is extended to 1e-1. It is case by case.